### DISCUSSION TITLE
When i reorganize the files fade out stops working

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#65] [Q&A]
When I reorganize the files the fade out stops working

**Comment by DimensionWarped** [comment]
Alright, I want you to think about this...

We don't know what you moved. We don't know where you moved it to. We don't know *which* fade out you are referring to. If you want help with this, you need to provide either a link to your branch where you are seeing this or else a diff file that reproduces what you are seeing against our primary active branch. You also need to provide a very specific description of the issue... as in, what kind of transition are you expecting a fade out for and what does it look like right now, what did it look like before you moved files around.

Obviously moving files around can cause anything that depends on those files to break. You'll need to figure out which references by path that you've broken and change them to match the new paths. The easiest way to do that is to look through your differences (use git diff) and do some searches for anything with a '/' in it.
